stubs/paramiko/paramiko/channel.pyi

```diff
@@ -41,14 +41,14 @@ class Channel(ClosingContextManager):
     def __init__(self, chanid: int) -> None: ...
     def __del__(self) -> None: ...
     def get_pty(
-        self, term: str = ..., width: int = ..., height: int = ..., width_pixels: int = ..., height_pixels: int = ...
+        self, term: bytes = ..., width: int = ..., height: int = ..., width_pixels: int = ..., height_pixels: int = ...
```

Comment:
The default is a `str`:

https://github.com/paramiko/paramiko/blob/a598ca14d6451343c69be97e0e1c65eb15c0e607/paramiko/channel.py#L166

As this is passed to `add_string()`, which accepts both `str` and `bytes`, I suggest to use:

```suggestion
        self, term: str | bytes = ..., width: int = ..., height: int = ..., width_pixels: int = ..., height_pixels: int = ...
```